src/WrapperApp/components/NavDrawer/NavDrawer.test.ts

```diff
@@ -19,20 +19,33 @@ describe('NavDrawer component', () => {
 	//this test checks if the menu button (upper left corner, next to the yaptide logo) has rendered
 	test('renders a menu button', async () => {
 		await driver.get('http://localhost:3000');
+
+		//hide react error overlay
```

Comment:
why do you need to hide react error overlay ?

@tudde have you checked documentation of mui , namely migration guide from v5 to v6 ?

https://mui.com/x/migration/migration-tree-view-lab/

Reply:
@grzanka this is done in order to prevent tests failing locally. This also happens on master - some time ago a completely unrelated error started appearing and it's interfering with selenium tests, as it's a dom element displayed over the actual application. This could prove useful for local testing without turning the overlay off each time.

Reply:
Can you make this "hiding feature" optional an parametrise the tests so the hiding happens only when running locally ? It's very easy to break the test behaviour with such changes. By default (on CI) nothing should be hidden.